backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/EnumSpecialDescriptorsFactory.kt

```diff
@@ -53,7 +54,7 @@ internal class EnumSpecialDeclarationsFactory(val context: Context) {
         val endOffset = enumClassDescriptor.endOffsetOrUndefined
 
         val implObjectDescriptor = ClassDescriptorImpl(enumClassDescriptor, "OBJECT".synthesizedName, Modality.FINAL,
-                ClassKind.OBJECT, listOf(context.builtIns.anyType), SourceElement.NO_SOURCE, false)
+                ClassKind.OBJECT, listOf(context.builtIns.anyType), SourceElement.NO_SOURCE, false, LockBasedStorageManager.NO_LOCKS)
```

Comment:
Would it potentially affect using compiler in IDE context?

Reply:
not sure.